doc/examples/kinematics_cost_function/launch/kinematics_cost_function_tutorial.launch.py

```diff
@@ -0,0 +1,21 @@
+from launch import LaunchDescription
+from launch_ros.actions import Node
+from moveit_configs_utils import MoveItConfigsBuilder
+
+
+def generate_launch_description():
+    moveit_config = MoveItConfigsBuilder("moveit_resources_panda").to_moveit_configs()
+
+    move_group_demo = Node(
+        name="kinematics_cost_fn_tutorial",
+        package="moveit2_tutorials",
+        executable="kinematics_cost_function_tutorial",
+        output="screen",
+        parameters=[
+            moveit_config.robot_description,
+            moveit_config.robot_description_semantic,
+            moveit_config.robot_description_kinematics,
```

Comment:
I believe this should fix it

```suggestion
            moveit_config.robot_description_kinematics,
            moveit_config.joint_limits,
```